immersive-web-wg-charter.html

```diff
@@ -300,7 +300,8 @@ <h3>
           </dl>
 
           <p>
-            The Working Group also may choose to deliver normative specifications for the following features, after sufficient incubation in the Community Group.
+            The Working Group also may choose to deliver normative specifications for the following features, if there is consensus in the Working Group that they are ready to move to the Recommendation track after sufficient incubation in the <a href="https://www.w3.org/community/immersive-web/">Immersive Web Community Group</a>.
+            If additional normative specifications need to be added to the Charter before the Charter expires, the Working Group will recharter with changes.
```

Comment:
I would not limit ourselves with this line.  I would rather remove it.

Reply:
updated in 5c6da10 as comment.